src/spec-node/dockerCompose.ts

```diff
@@ -704,4 +704,22 @@ function convertMountToVolume(mount: Mount): string {
 	volume += mount.target;
 
 	return volume;
-}
\ No newline at end of file
+}
+
+/**
+ * Convert mount command' arguments to volume top-level element
+ * @param mount 
+ * @returns mount object representation as volumes top-level element
+ * @throws if `source` property is undefined
+ */
+function convertMountToVolumeTopLevelElement(mount: Mount): string {
+	let volume: string = `
+  ${mount.source}
+`;
```

Comment:
nit: This seems to include an extra newline compared to before.

Reply:
Fixed in d2510412db8afb196ab6e8f852021de2338d8c30